tests/e2e-pw/specs/merchant/merchant-orders-manual-capture.spec.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { getShopper, getMerchant } from '../../utils/helpers';
+import {
+	goToOrder,
+	goToWooPaymentsSettings,
+} from '../../utils/merchant-navigation';
+import { saveWooPaymentsSettings } from '../../utils/merchant';
+import {
+	emptyCart,
+	fillCardDetails,
+	focusPlaceOrderButton,
+	placeOrder,
+	setupProductCheckout,
+} from '../../utils/shopper';
+import { goToShop } from '../../utils/shopper-navigation';
+
+/**
+ * Local variables.
+ */
+let orderId;
+
+test.describe( 'Order > Manual Capture', () => {
+	test.beforeEach( async ( { browser } ) => {
+		// Merchant go to settings, enable capture later, and then save.
+		const { merchantPage } = await getMerchant( browser );
+		await goToWooPaymentsSettings( merchantPage );
+		await merchantPage.getByTestId( 'capture-later-checkbox' ).click();
+		await merchantPage
+			.getByRole( 'button', { name: 'Enable manual capture' } )
+			.click();
+		await saveWooPaymentsSettings( merchantPage );
+
+		// Shopper add items to cart, fill in the checkout, place an order.
+		const { shopperPage } = await getShopper( browser );
+		await emptyCart( shopperPage );
+		await goToShop( shopperPage, 1 );
+		await setupProductCheckout( shopperPage );
+		await fillCardDetails( shopperPage );
+		await focusPlaceOrderButton( shopperPage );
+		await placeOrder( shopperPage );
+
+		// Confirm that the order was placed and get the order number.
+		await shopperPage.waitForURL( /\/order-received\//, {
+			waitUntil: 'load',
+		} );
+		await expect(
+			shopperPage.getByRole( 'heading', { name: 'Order received' } )
+		).toBeVisible();
+		const orderIdField = shopperPage.locator(
+			'.woocommerce-order-overview__order.order > strong'
+		);
+		orderId = await orderIdField.innerText();
```

Comment:
This can be greatly simplified by using the `placeOrderWithOptions` helper 🙂 

```ts
orderId = await placeOrderWithOptions( shopperPage );
```

Reply:
Thank you, I think I had tried using that function and something about it didn't work, so I found where an order was placed elsewhere and copied that with modifications for my needs. I did try it now and it worked, so updated in https://github.com/Automattic/woocommerce-payments/pull/10180/commits/22412db0728f4d4a1b443fab238494f237bbdb39